src/Core_Command.php

```diff
@@ -152,8 +155,20 @@ public function download( $args, $assoc_args ) {
 
 		$locale       = Utils\get_flag_value( $assoc_args, 'locale', 'en_US' );
 		$skip_content = Utils\get_flag_value( $assoc_args, 'skip-content' );
+		$from_url     = false;
 
-		if ( isset( $assoc_args['version'] ) && 'latest' !== $assoc_args['version'] ) {
+		if ( ! empty( $args[0] ) ) {
```

Comment:
We can directly check for `$from_url` now, which makes the conditional clearer and the following comment redundant.
```suggestion
		if ( $from_url ) {
```